src/CoolPropLib.cpp

```diff
@@ -604,6 +604,43 @@ EXPORT_CODE void CONVENTION AbstractState_get_mole_fractions_satState(const long
         HandleException(errcode, message_buffer, buffer_length);
     }
 }
+// TODO: verify this is correct
+EXPORT_CODE double CONVENTION AbstractState_get_fugacity(const long handle, const std::size_t i, long* errcode, char* message_buffer,
+                                                         const long buffer_length) {
+    *errcode = 0;
+    try {
+        shared_ptr<CoolProp::AbstractState>& AS = handle_manager.get(handle);
+        return AS->fugacity(i);
+    } catch (...) {
+        HandleException(errcode, message_buffer, buffer_length);
+    }
+    return _HUGE;
+}
+// TODO: verify this is correct
+EXPORT_CODE double CONVENTION AbstractState_get_fugacity_coefficient(const long handle, const std::size_t i, long* errcode, char* message_buffer,
+                                                                    const long buffer_length) {
+    *errcode = 0;
+    try {
+        shared_ptr<CoolProp::AbstractState>& AS = handle_manager.get(handle);
+        return AS->fugacity_coefficient(i);
+    } catch (...) {
+        HandleException(errcode, message_buffer, buffer_length);
+    }
+    return _HUGE;
+}
+// TODO: verify this is correct
+// Am I returning vectors correctly??
+// EXPORT_CODE vector<std::double> CONVENTION AbstractState_get_fugacity_coefficients(const long handle, const std::size_t i, long* errcode, char* message_buffer,
```

Comment:
I would say let's leave this out until implemented throughout CoolProp

Reply:
Ok, I will remove the commented block